plugin/web.py

```diff
@@ -47,27 +46,38 @@ def advance(self, recvSize=1024):
             return False
 
         rlist, wlist = select.select([self.sock], [self.sock], [], 0)[:2]
+        self.sock.settimeout(1.0)
```

Comment:
This seems to be a bit aggressive; I agree that this will work for most situations (especially considering that in 99.9% of the cases we will be connected to localhost), but it might still be good to bump this up to 5 seconds or so.

Reply:
I thought this slowed things down previously, but upon testing just now, it does seem like everything works without lag when increasing the timeout.